svd/esp32.base.svd

```diff
@@ -37746,7 +37746,2336 @@
                 <size>0x00000000</size>
```

Comment:
Should this be updated? It's not 100% clear to me what it means. I would assume the size of the mmio register block?